core/operation.ts

```diff
@@ -111,6 +119,11 @@ export class Operation {
       : dataform.ActionHermeticity.NON_HERMETIC;
   }
 
+  public disabled() {
```

Comment:
I wonder if we really want to encourage this syntax if the same can be achieved in the config block? I think at some point we should move to these APIs being fully object based, like:

```
publish({
  name: "x",
  query: ctx => ...
}
```

In other words I'd be tempted to leave this out for now

Reply:
I kinda agree, but OTOH i do think API consistency is king and it costs us very little to have these, so I'd prefer to keep them